### DISCUSSION TITLE
Shoelace-component not copying value

### DISCUSSION THREAD

**Discussion by stevan06v** [livewire/livewire#8116] [1. Help]
I am using shoelace for my prestyled-components in livewire and I tried using the component for the copy-button. The copy functionality does not work. There are no errors thrown in the browser console. 

Could this be due to livewire?

```
 <div class="flex gap-2 items-center">
            <sl-input wire:ignore id="project_hash" type="text" value="{{$project_hash}}" disabled></sl-input>
            <sl-copy-button wire:ignore class="text-2xl text-primary-blue" from="project_hash.value"></sl-copy-button>
        </div>
```

**Comment by joshhanley** [comment]
@Stevan06v it looks like you don't have a `wire:click` on your button, maybe that's causing the issue?

**Reply by stevan06v** [reply]
The js behind the custom shoelace component handles the click and the copy-functionality. Is a wire:click neccessary? I am not using livewire for the copying logic :)

**Reply by joshhanley** [reply]
@Stevan06v ah right, never used it so I'm not sure. Maybe as a test try moving the wire:ignore to the parent div.
